lessons/operator/01/README.md

```diff
@@ -1,122 +1,121 @@
+## Monitoring as Code
 
-> Everything as code initiatives as we know them are built on top of **pipelines that enable organizations to codify complex business requirements as repeatable steps**.
-> The CI pipeline (build+test) paved the way for Continuous Delivery (CD).
-> Together as the CI/CD pipeline (build+test, deploy), they provided the foundation for infrastructure as code (build+test, provision+test, deploy).
-> Like IaC, monitoring as code hooks into this CI/CD pipeline, but also requires a dedicated pipeline for ongoing observation of our systems (build+test, provision+test, deploy, observe).
-> We call this missing link the observability pipeline.
->
-> ![](img/everything-as-code.png)
+There is a strong movement towards managing all aspects of the software industry in the same way that we manage application code. 
+Collectively these are referred to as "everything-as-code" methods.
 
-Simply shoving observability data through a generic pipeline where you can cobble together disparate components to filter and route data to various backends is a great first step in an observability pipeline journey – but how to you manage the overall solution?
-And how do you onboard new users to such a bespoke solution – including teams with a multitude of disparate tools spanning multiple generations of technology?
-Sensu's turn-key observability pipeline provides a centralized control plane and end-to-end solution for monitoring as code, which enables organizations to onboard workloads to the observability pipeline in an incremental manner.
+Sensu is designed specifically to enable this kind of workflow for monitoring.
 
-![](img/technology-adoption-lifecycle.jpg)
+### Sensu is an API, Configurations are YAML
 
-_FIGURE 3: the "as code" technology adoption lifecycle._
+The Sensu _backend_ is a service that provides the API and performs event processing.
+All aspects of the observability pipeline can be modified at runtime through the API. 
+Operators can add new components, modify the behavior of filters and handlers, silence noisy streams of data, or add new database sinks *at any time*, without the need to re-deploy.
 
+Components are defined in YAML files, which are pushed to the API to build the pipeline.
+These YAML files can be stored in source-control alongside your application code, and can be tested, versioned, and managed like any other code file.
+This workflow takes the burden off of system operators, by allowing engineers to self-service their monitoring configurations, with security and access controls at every layer.
 
-## Use cases
+To learn more, please check out our [monitoring as code whitepaper](https://sensu.io/resources/whitepaper/monitoring-as-code-with-sensu).
 
-Sensu's observability pipeline
+### Bonsai Asset Repository
 
-- **Eliminate data silos**
+Another essential component of Sensu is the [Bonsai Asset Repository](https://bonsai.sensu.io/).
```

Comment:
Proposing a rewrite of this entire paragraph; please feel free to slice & dice:

> Another essential component of Sensu is the [runtime asset](https://docs.sensu.io/sensu-go/latest/plugins/assets/) – shareable, reusable packages that make it easy to deploy monitoring plugins across heterogenous infrastructure. 
> Sensu Assets work with bare metal servers, cloud compute instances, and containers, and they are compatible with Linux, Windows, MacOS, and other Unix operating systems. 
> 
> Sensu hosts a public repository of commercially- and community-supported assets called [Bonsai](https://bonsai.sensu.io). 
> This service facilitates real-time distribution of assets to agents and backends based on the monitoring configuration defined in the Sensu API. 
> In this way, operators and developers can focus on the monitoring configuration without worrying about how to security distribute monitoring plugins to multi-cloud infrastructure.
> 
> There are hundreds of open-source plugins available on Bonsai, supporting nearly every imaginable integration.
> [Have a look around](https://bonsai.sensu.io/assets)! 
> If what you need isn't already available, the [Sensu Plugin SDK](https://github.com/sensu/sensu-plugin-sdk) makes it easy to develop and publish new custom plugins.

Reply:
Yep. This is exactly where my mind was going for this rewrite. Will lightly edit this and replace. As written it's a bit too long.